{{cookiecutter.project_slug}}/.circleci/config.yml

```diff
@@ -105,8 +107,9 @@ jobs:
     steps:
       - add_ssh_keys:
           fingerprints:
-
-            - "c5:bd:6d:0b:a1:39:97:83:ca:b5:13:59:14:ff:49:40"
+{%- endraw %}
+            - "{{ cookiecutter.ssh_key_fingerprint }}"
+{% raw -%}
```

Comment:
The SSH key fingerprint shouldn't be hardcoded in case it needs to be changed.

Reply:
Just double checked:
https://github.com/crowdbotics/crowdbotics-slack-app/blob/master/scaffold/mobile_app/react_native/scaffold.py#L27

And its also set in staging/prod env vars